onboard-blink/cpp/README.md

```diff
@@ -1,872 +1,31 @@
+# Onboard Blink 
 
 
+## Introduction
+This is a simple sample you could use for a quick test.
 
+## What it is
+A blink sample, it turns the on-board LED on for one second, then off for one second, repeatedly.
 
+## Hardware requirements
 
```

Comment:
This additional blank line should be hear? the other titles do not have a blank line after it

Reply:
Or we should add the blank line after the other titles as well.